example/scalalib/native/4-common-config/src/Foo.scala

```diff
@@ -0,0 +1,24 @@
+package foo
+
+import scala.scalanative.libc._
+import scala.scalanative.unsafe._
+import fansi._
+
+object Foo {
+
+  def generateHtml(text: String): CString = {
+    val colored = Console.RED + "<h1>" + text + "</h1>" + Console.RESET
+
+    implicit val z: Zone = Zone.open
+    val cResult = toCString(colored)
+    z.close()
+    cResult
+  }
```

Comment:
What is the point of this code?
it seems just scala code converted to C and also used after `free`. You can't use the CString after closing the Zone.
A better example would be using an open source html templating c library and then convert the C result to a Scala string

Reply:
It was meant to be a strawman hello-world Scala-Native application, so intentionally overly simple. Is it easy to integrate open source C libraries into scala native? I've never done it before